Makefile

```diff
@@ -447,6 +447,13 @@ test-ui:
 test-e2e-ci:
 	cd ui; npm run test:e2e:ci; cd ..
 
+#-----------------------------------------------------------------------------
+# Rego Policies
+#-----------------------------------------------------------------------------
+rego-eval:	
```

Comment:
@leecalcote @Jougan-0 

This is an attempt to create some initial tooling to help contributors work with Rego policies. It's not a perfect solution but maybe acts as a starting place until we get some more robust testing in place? Let me know what you think.

Reply:
I wonder if we might rename "input.json" to the name of a design file. I seems that an ideal design file might be a copy of the "All Relationships" design from the catalog - https://meshery.layer5.io/catalog/content/design/all-relationships-7b717cb2-24a1-4f4a-976e-ef1c9a982696?source=%257B%257D

Reply:
Thank you for following precedent and using "noun-verb" in the naming of this make target. 👏